demo/ARRoute+DirectionsRoute.swift

```diff
@@ -28,3 +28,46 @@ extension MapboxVisionARNative.Route {
                   destinationStreetName: route.legs.last?.destination.name ?? "")
     }
 }
+
+func convert(maneuverType: MapboxDirections.ManeuverType) -> MapboxVisionARNative.ManeuverType {
```

Comment:
WDYT about an extension on `MapboxDirections.ManeuverType` for this?

Reply:
changed